zenodb.go

```diff
@@ -36,13 +36,10 @@ const (
 	DefaultIterationCoalesceInterval = 3 * time.Second
 	DefaultIterationConcurrency      = 2
 
-	DefaultClusterQueryConcurrency = 25
-	DefaultClusterQueryTimeout     = 1 * time.Hour
+	DefaultClusterQueryTimeout = 1 * time.Hour
 )
 
 var (
-	log = golog.LoggerFor("zenodb")
```

Comment:
The logger is now a property on the database so that we can include the database type and id as part of our log messages.